packages/logger/src/logger/Logger.ts

```diff
@@ -121,18 +126,28 @@ function resumeLogQueueProcessing(transports: Transport[]): void {
   }
 }
 
+export function generateRandomRunId() {
+  return Math.round(Math.random() * Number.MAX_SAFE_INTEGER).toString();
```

Comment:
I'm curious, why did we opt for a random number to generate the run ID? I was thinking of using `uuidv4()` to ensure uniqueness. Personally, my brain tends to recognize strings like `"f47ac10b-58cc-4372-a567-0e02b2c3d479"` as IDs more easily than large numbers, but that’s just my preference :D

Reply:
Good call. No real reason, was just the first thing I thought of. I don't want to add a new package, but nodejs has a require("crypto").randomUUID() that I think should do the job.

Reply:
Done.